### DISCUSSION TITLE
Do you use the Export for cluster computation/Import PK-Analyses features in PK-Sim?

### DISCUSSION THREAD

**Discussion by msevestre** [Open-Systems-Pharmacology/Forum#1214] [Polls]
Dear community,

PK-Sim supports the concept of offloading the calculation of a population simulation to another engine (a cluster maybe) and loading the simulation results back into the application. This feature is called **Export for cluster computation** and was implemented at a time when the scripting capabilities of the OSPSuite were very limited. With v9, the OSPSuite started to offer a very powerful interface with R (OSPSuite-R) which rendered this feature irrelevant. 

![image](https://user-images.githubusercontent.com/1041237/169834031-900a2a70-53cc-4820-8e99-ab3394687101.png)

The code required to maintain this feature is fairly complicated and we would like to remove it entirely. 

The same goes with the **Import PK-Analyses** feature where a user could potentially calculate some user-defined PK-Parameters outside of PK-Sim and import them for analyses. OSPSuite-R now fully supports user-defined PK-Parameters 

Please select **Yes** in the poll below if you use at least one of these features regularly (e.g more than once in the last 8 years). If you select yes, don't hesitate to leave a message explaining the use case.
Otherwise, please select **No**

Thanks for your feedback

**Comment by Incei** [comment]
I have used import PK-Analyses some years ago after exporting, adjusting and re-importing back into PK-Sim. I have not used it for a long time, but it was quite useful. If the R interface renders this feature irrelevant, I don't see why we would need double features.

**Comment by PavelBal** [comment]
I can image that "Import PK-Analyses..." is still a useful feature. You could do PK-calculation in R but import in PK-Sim in order to visualize and compare.

**Comment by RobinM92** [comment]
I have not used this feature but can see the merit of being able to run simulations, estimations or sensitivity analyses (partly) on a cluster. Especially more complex mobi models quickly become computationally expensive. With that in mind, I feel that a good guide of what is exactly possible with the osp suit r package would be very beneficial, I hear from many that they try to use it for their work but get turned off by the lack of documentation. Just as a small feedback to this user poll, thank you for the amazing work!

**Reply by msevestre** [reply]
> I hear from many that they try to use it for their work but get turned off by the lack of documentation

This is very surprising. We have a full website documenting the tools
https://www.open-systems-pharmacology.org/OSPSuite-R/

specifically
![image](https://user-images.githubusercontent.com/1041237/169851322-a2d88026-37b7-4407-92bf-97ac39a817b6.png)
